Formula/qt.rb

```diff
@@ -203,6 +204,7 @@ def install
     if OS.mac?
       cmake_args << "-DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}.0"
       config_args << "-sysroot" << MacOS.sdk_path.to_s
+      config_args << "-system-harfbuzz"
```

Comment:
Why not on linux?

Reply:
Does it break if we still pass this flag?

Reply:
Can we unconditionally pass this then? It makes it easier to see the state of the dependency.